src/utils/pool.rs

```diff
@@ -49,6 +52,12 @@ pub struct Pool<VS: VersionSet, N: PackageName = String> {
     /// Map from version set to the id of their interned counterpart
     version_set_to_id: FrozenCopyMap<(NameId, VS), VersionSetId, ahash::RandomState>,
 
+    /// Conditions that can be used to filter solvables.
+    conditions: Arena<ConditionId, Condition>,
+
+    /// Map from condition to its id
+    pub(crate) condition_to_id: FrozenCopyMap<Condition, ConditionId, ahash::RandomState>,
```

Comment:
is this pub crate needed?

Reply:
I just kinda copied it from other fields, but probabbly it's not needed. Let me remove it for now.

<img width="786" height="526" alt="Image" src="https://github.com/user-attachments/assets/bd01da55-fbfd-4dba-9122-110e4465e5ce" />